content/en/docs/2023.11/getting-started/on-premise/install-innovation-only/multiple-server-with-ha/install-web-application-server/perform-gateway-installation.md

```diff
@@ -181,7 +31,7 @@ To install the components required for debugging, perform the steps detailed in
     -ServiceFabricUsingSelfSignedCertificates $false `
     -ServiceFabricApiGatewayBasicAuthUsername "BasicAuthUser" `
     -ServiceFabricApiGatewayBasicAuthPassword '#_065077199197085!212123173135087074174142102155007175102029143220132038175026114248243207204119030125106032237087162060168108135168241247037070081~187087056217118!069132229129134129097089241180163#' `
-    -DotNetFlowDebuggerEndpoint "https://server.domain.com:8722/api/" `
+    -DotNetFlowDebuggerEndpoint 'https://server.domain.com:8722/api/' `
```

Comment:
do we need to do that for all params?

Reply:
no only passwords should be single quotes - have replaced this one with double